### ISSUE TITLE
Add "interesting websites"

### ISSUE THREAD

**Issue by Neikon** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#35]
I think it would be interesting to add some resources to the website to make it more complete. We could add them to the footer for example.

Resources:
https://github.com/Timvde/UserChrome-Tweaks
https://www.userchrome.org/
Tool to inspect the UI: https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox

What do you think?

**Comment by BeyondMagic** [comment]
Yeah, it will help users who like full customizations, but Timvde's repository seems not maintained anymore, unfortunately, do you think we should add it?

**Comment by Neikon** [comment]
> 
> 
> Yeah, it will help users who like full customizations, but Timvde's repository seems not maintained anymore, unfortunately, do you think we should add it?

the last commit was in december, maybe it is not completely dead. and even if it is unmaintained I think it has a lot of settings that can be useful to customize in some aspects other themes. Assuming that most of the things that are there still work, which I guess they do.

**Comment by andreasgrafen** [comment]
I like the idea. We can also link back to r/firefoxcss on that site. While most people probably know about it anyway it's a nice link for those who might not. ^—^
